includes/polling/processors/hpmsm.inc.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2017 Søren Friis Rosiak <sorenrosiak@gmail.com>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$proc = snmp_get($device, 'coUsInfoCpuUseNow.0', '-Ovq', 'COLUBRIS-USAGE-INFORMATION-MIB');
```

Comment:
I don't think you need this file at all if you register `discover_processor()` with a numerical oid